templates/login.liquid

```diff
@@ -161,75 +96,15 @@
         }
       }
     </style>
-    <script>
-      function hideForumDetails() {
-        const details = document.getElementById("forum-instructions-details");
-        details.classList.remove('open');
-        details.classList.add('closed');
-        const showMore = document.getElementById("forum-instructions-show-more");
-        showMore.classList.remove('closed');
-        showMore.classList.add('open');
-      }
-      function showForumDetails() {
-        const details = document.getElementById("forum-instructions-details");
-        details.classList.remove('closed');
-        details.classList.add('open');
-        const showMore = document.getElementById("forum-instructions-show-more");
-        showMore.classList.remove('open');
-        showMore.classList.add('closed');
-      }
-    </script>
```

Comment:
Did we even use this script? Weird.

Reply:
Yes, but the "show more" button was only visible on mobile.